server/base/src/main/java/org/apache/accumulo/server/security/handler/ZKPermHandler.java

```diff
@@ -272,11 +268,9 @@ public void revokeSystemPermission(String user, SystemPermission permission)
 
     try {
       if (sysPerms.remove(permission)) {
-        synchronized (zooCache) {
-          zooCache.clear();
-          zoo.putPersistentData(zkUserPath + "/" + user + ZKUserSysPerms,
-              ZKSecurityTool.convertSystemPermissions(sysPerms), NodeExistsPolicy.OVERWRITE);
-        }
+        ctx.getZooCache().clear((path) -> path.startsWith(sysPermPath));
```

Comment:
Not a change for this PR, but it seems like these clears could be removed from the code.  The cache should eventually update as an event will be generated from the zookeeper update.  Its nice to put all the paths that used more than once in a variable in this PR.